src/server/server.js

```diff
@@ -394,4 +404,58 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function localizeHtmlFile(htmlFile, html, lang) {
+    // Replace localized strings
+    var targetJsonFile = getI18nJsonFile(htmlFile, lang);
+    if (utils.existsSync(targetJsonFile)) {
+        try {
+            var locItems = require(targetJsonFile);
```

Comment:
Oh yeah, good point (I rearranged this at the last minute and forgot to update that).